coding/javascript.md

```diff
@@ -23,88 +23,85 @@ On all publicly accessible websites, core content and functionality should still
 
 ## React
 
 ### Starting a new project
 
-When starting a new react application, we recommend [create-react-app](https://github.com/facebook/create-react-app) for bootstrapping the project. Avoid [ejecting](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject) as long as feasible to make updating dependencies easier.
+When starting a new React application, we recommend [create-react-app](https://github.com/facebook/create-react-app) for bootstrapping the project. Avoid [ejecting](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject) as long as feasible to make updating dependencies easier.
 
 ### Preferred libraries
 
 All projects should generally use the following:
 
-* style framework - [vanilla](https://github.com/vanilla-framework/vanilla-framework)
-* component testing - [enzyme](https://github.com/airbnb/enzyme)
+* style framework - [Vanilla](https://github.com/vanilla-framework/vanilla-framework)
+* component testing - [Enzyme](https://github.com/airbnb/enzyme)
 
 If you require routing, or state management:
 
 * routing - [react-router](https://github.com/ReactTraining/react-router)
-* app state management - [redux](https://redux.js.org)
+* app state management - [Redux](https://redux.js.org)
 
 #### Redux specific
 
-* selector composition and memoisation - [reselect](https://github.com/reduxjs/reselect)
+* selector composition and memoisation - [Reselect](https://github.com/reduxjs/reselect)
 * async/side-effects management (http, localstorage etc.) - [redux-saga](https://github.com/redux-saga/redux-saga)
 
 If you'd like introduce a new library, or feel we should replace one of the above, please make a PR to start a discussion.
 
 ### Reference projects
 
-[CRBS-UI](https://git.launchpad.net/~crbs/crbs/+git/crbs-ui/tree/), while under active development, probably best reflects our standards for react projects currently.
+[CRBS-UI](https://git.launchpad.net/~crbs/crbs/+git/crbs-ui/tree/), while under active development, probably best reflects our standards for React projects currently.
 
-[build.snapcraft.io](https://github.com/canonical-websites/build.snapcraft.io) - a bit dated, but first fully React centred project in the webteam. Our first use of React, server-side rendering, Redux, Enzyme, etc. While it doesn't fully follow our current standards it may be a good place to check how some of these libraries were used.
+[build.snapcraft.io](https://github.com/canonical-websites/build.snapcraft.io) - a bit dated, but first fully React centered project in the webteam. Our first use of React, server-side rendering, Redux, Enzyme, etc. While it doesn't fully follow our current standards, it may be a good place to check how some of these libraries were used.
```

Comment:
We actually use British English, since we're a  UK-based company. "centred" is the correct form to conform with our [copy guidelines](https://canonical-webteam.github.io/practices/content/copy-reviews.html).

```suggestion
[build.snapcraft.io](https://github.com/canonical-websites/build.snapcraft.io) - a bit dated, but first fully React-centred project in the webteam. Our first use of React, server-side rendering, Redux, Enzyme, etc. While it doesn't fully follow our current standards, it may be a good place to check how some of these libraries were used.
```